src/JestExt.ts

```diff
@@ -450,12 +451,11 @@ export class JestExt {
 
   private shouldIgnoreOutput(text: string): boolean {
     // this fails when snapshots change - to be revised - returning always false for now
-    return text.includes('Watch Usage')
+    return text.includes('Watch Usage') || text.includes('onRunComplete') || text.includes('onRunStart')
   }
 
   private testsHaveStartedRunning() {
     this.channel.clear()
-    this.status.running('initial full test run')

```

Comment:
you probably don't this method anymore. But it would be nice to keep the 'initial full test run' message, which I believe  can be done at  `onRunStart` by examing the jestProcess's watchMode.

Reply:
I completely remove `testsHaveStartedRunning()` method as it is no longer required. In my opinion "initial full test run" message was also a bit redundant as just after the start it was replaced by "Running tests" message. Please let me know how do you like that approach. I just prefer a simpler code over another message in the status bar.